src/cirrus/core/utils/plugins.py

```diff
@@ -1,15 +1,16 @@
+from collections.abc import Generator
+
 try:
-    from importlib.metadata import entry_points
+    from importlib.metadata import EntryPoint, entry_points
 except ImportError:
-    from importlib_metadata import entry_points
-
+    from importlib_metadata import EntryPoint, entry_points
 
 PLUGIN_GROUP = "cirrus.plugins"
 COMMANDS_GROUP = "cirrus.commands"
 RESOURCES_GROUP = "cirrus.resources"
 
 
-def iter_entry_points(group_name):
+def iter_entry_points(group_name: str) -> Generator[EntryPoint]:
```

Comment:
Per the docs, I don't believe this to be a valid generator type annotation: https://docs.python.org/3/library/typing.html#typing.Generator. I think it either needs to be `Generator[EntryPoint, None, None]` or `Iterator[EntryPoint]`.

Reply:
Correct you are.